partialWaveFit/pwaLikelihood.h

```diff
@@ -180,9 +182,13 @@ namespace rpwa {
 		bool addAccIntegral(rpwa::ampIntegralMatrix& accMatrix, unsigned int accEventsOverride = 0);
 
 		bool addAmplitude(const rpwa::amplitudeMetadata& meta);
```

Comment:
Could you please remove this version of the interface and propagate that throughout the project.